### PR TITLE
[BUG] fix added buttons

### PR DESCRIPTION
**Pull request by kishoreravi24**
[BUG] The tags on the search page should be buttons #7037

## Fixes Issue
- Tags on the search page added as div now replaced with button

## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Screenshots

after_button_changes

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/7096"></a>

### CODE REVIEW

**Review by AliMora83 — COMMENTED** [review]
Very good 🥳

File: pages/search.js [hunk 1]
```diff
@@ -9,6 +9,7 @@ import Tag from "@components/Tag";
 import Badge from "@components/Badge";
 import logger from "@config/logger";
 import Input from "@components/form/Input";
+import Button from "@components/Button";
```
Comment (eddiejaoude) [inline pages/search.js:12]: This import is not used?

File: pages/search.js [hunk 2]
```diff
@@ -136,7 +137,7 @@ export default function Search({ data }) {
       <Page>
         <h1 className="text-4xl mb-4 font-bold">Search</h1>
 
-        <div className="flex flex-wrap justify-center space-x-3 mb-4">
+        <button className="flex flex-wrap justify-center space-x-3 mb-4">
```
Comment (eddiejaoude) [inline pages/search.js:140]: This is just 1 button? Each tag needs to be its own button

**Review by eddiejaoude — CHANGES_REQUESTED** [review]
Thank you for your contribution. Please see inline comments

File: pages/search.js (hunk 1 above)
Comment (kishoreravi24) [reply pages/search.js:12]: removed unused uimport

File: pages/search.js (hunk 2 above)
Comment (kishoreravi24) [reply pages/search.js:140]: The tags are being mapped over and rendered as Tag components with button elements, If there are 10 tags available the code will render only the available tags as buttons, so we can have 10 buttons, same logic as div with replace of div we are using buttons now.

Comment (ChinmayMhatre) [reply pages/search.js:139]: @kishoreravi24 this div is just a wrapper . The `div` inside the `Tag` component needs to be changed to a button. This needs to remain a `div`

**Review by ChinmayMhatre — DISMISSED** [review]

File: pages/search.js (hunk 2 above)
Comment (kishoreravi24) [reply pages/search.js]: @ChinmayMhatre thanks for the explanation, done fixed the cc comments please check now 👍

**Comment by kishoreravi24** [comment]
Hi @eddiejaoude any update on this PR, Does the changes look good or need to change some?

**Review by eddiejaoude — APPROVED** [review]
Looks good, thank you for the contribution 👍 

Great collaboration everyone 💪

**Pull request merged**